<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="modificar_lineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="modificar_Widget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>681</width>
     <height>571</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;수정&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSearch">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>이름</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>아파트동호수</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>아이디</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>연락처</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labePlus">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>830</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="searchprev">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>840</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;이전</string>
   </property>
   <property name="icon">
    <iconset theme="화살표">
     <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="searchnext">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>840</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>다음&gt;</string>
   </property>
   <property name="icon">
    <iconset theme="화살표">
     <normaloff>../../.designer/backup</normaloff>../../.designer/backup</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonUpdate">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>최신등록순</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
